client/src/templates/Challenges/components/tool-panel.tsx

```diff
@@ -104,40 +101,26 @@ function ToolPanel({
           {isMobile ? t('buttons.reset') : t('buttons.reset-lesson')}
         </Button>
       )}
-      <DropdownButton
-        block={true}
-        bsStyle='primary'
-        className='btn-invert'
-        id='get-help-dropdown'
-        title={isMobile ? t('buttons.help') : t('buttons.get-help')}
-      >
-        {guideUrl ? (
-          <MenuItem
-            bsStyle='primary'
-            className='btn-invert'
-            href={guideUrl}
-            target='_blank'
-          >
-            {t('buttons.get-hint')}
-          </MenuItem>
-        ) : null}
-        {videoUrl ? (
-          <MenuItem
-            bsStyle='primary'
-            className='btn-invert'
-            onClick={openVideoModal}
-          >
-            {t('buttons.watch-video')}
+      <Dropdown dropup>
+        <Dropdown.Toggle dropup id={'get-help-dropdown'}>
+          {isMobile ? t('buttons.help') : t('buttons.get-help')}
+        </Dropdown.Toggle>
+        <Dropdown.Menu dropup>
+          {guideUrl ? (
+            <MenuItem href={guideUrl} target='_blank'>
+              {t('buttons.get-hint')}
```

Comment:
```suggestion
              {t('buttons.get-hint')}{' '}
              <FontAwesomeIcon icon={faExternalLinkAlt} />
              <span className='sr-only'>, {t('aria.opens-new-window')}</span>
```

We should definitely add the sr-only text. The icon would be nice but isn't as essential if you don't like it.